doc/sphinx/source/howto/execute.rst

```diff
@@ -228,6 +223,7 @@ Put it all together
 
 .. literalinclude:: ../../../examples/abc.cpp
    :language: cpp
+   :linenos:
    :caption: "The (a + b) * c example for executing a computation on nGraph"
```

Comment:
Just curious: Is this just for consistency, or does it make some change to how things work?

Reply:
No change to how things work, just helps for anybody who wants to do a "view source" of the page.